Sources/Sentry/SentryReplayRecording.m

```diff
@@ -0,0 +1,64 @@
+#import "SentryReplayRecording.h"
+#import "SentryDateUtil.h"
+
+@implementation SentryReplayRecording
+
+- (instancetype)initWithSegmentId:(NSInteger)segmentId
+                             size:(NSInteger)size
+                            start:(NSDate *)start
+                         duration:(NSTimeInterval)duration
+                       frameCount:(NSInteger)frameCount
+                        frameRate:(NSInteger)frameRate
+                           height:(NSInteger)height
+                            width:(NSInteger)width
+{
+    if (self = [super init]) {
+        self.segmentId = segmentId;
+        self.size = size;
+        self.start = start;
+        self.duration = duration;
+        self.frameCount = frameCount;
+        self.frameRate = frameRate;
+        self.height = height;
+        self.width = width;
+    }
+    return self;
+}
+
+- (nonnull NSArray<NSDictionary<NSString *, id> *> *)serialize
+{
+
+    long timestamp = [SentryDateUtil millisecondsSince1970:self.start];
+
+    NSDictionary *metaInfo = @{
```

Comment:
note: we only have to send this with the first segment, and whenever the viewport/orientation changes, maybe leave a todo to not forget?

Reply:
I was not aware of it. 
Is it a problem to send it with every recording tho?

Reply:
shouldn't be